packages/block-library/src/navigation-menu-item/edit.js

```diff
@@ -25,6 +25,7 @@ import {
 	ENTER,
 } from '@wordpress/keycodes';
 import { __ } from '@wordpress/i18n';
+import { RichText } from '@wordpress/editor';
```

Comment:
E2E tests are failing because of a deprecation here.

```suggestion
import { RichText } from '@wordpress/block-editor';
```

Reply:
Thanks!